src/styles/styleRules.ts

```diff
@@ -173,8 +173,19 @@ export class StyleRules {
                 styleRule.input[1]
               } ${StyleRules.sanitizeValue(styleRule.type, styleRule.value)}`
             : `${StyleRules.getTypeLabel(styleRule.itemType)}`;
-          const value = styleRule.style.image;
-          StyleRules.updateLegend(result, {label: label, value: value});
+          const imageStyle = styleRule.style.image as IStyleImage;
```

Comment:
added a special case when the image URL style rule refers to a node property path.